src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -23,20 +24,21 @@ public class AddCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
             + "Parameters: "
             + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
```

Comment:
Should we include the optional fields in MESSAGE_USAGE as well?  like phone, email, address
Since we include tags as well